lib/elixir/lib/macro.ex

```diff
@@ -354,6 +354,22 @@ defmodule Macro do
 
   Checks the `type:Macro.t` for the specification of a valid
   quoted expression.
+
+  It returns `:ok` if the expression is valid. Otherwise it returns a tuple of the form
+  `{:error, remainder}` where invalid `remainder` of the quoted expression.
```

Comment:
actually, i think "in the format `{:error, remainder}`" is a good option as well